numbers.qmd

```diff
@@ -273,12 +273,12 @@ starting <- 100
 interest <- 1.05
 
 money <- tibble(
-  year = 2000 + 1:50,
```

Comment:
In this section the name "year" has been used in two different ways: a) the number of years interest is compounding (1:50), and b) in the form of the calendar year staring from 2000. For a) see here: "That gives a formula like money = starting * interest ^ year" ; and for b) the variable name "year" in the tibble. I think it would be more clear to only apply one meaning to the name "year" in this example.